device-types/Lenovo/system_x3250_M6.yaml

```diff
@@ -0,0 +1,21 @@
+---
+manufacturer: Lenovo
+model: System x3250 M6
+slug: system_x3250_m6
+part_number: 3633
+u_height: 1
+is_full_depth: true
+comments: '[Lenovo System x3250 M6 product guide](https://lenovopress.lenovo.com/lp0096.pdf)'
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c14
+  - name: PSU 2
+    type: iec-60320-c14
+interfaces:
+  - name: Ethernet 1
+    type: 1000base-t
+  - name: Ethernet 2
+    type: 1000base-t
+console-ports:
+  - name: Rear VGA
```

Comment:
VGA ports are not meant to be modelled this way, lets remove the console-ports/vga adapters

Reply:
removed